imod/mf6/simulation.py

```diff
@@ -1378,8 +1378,5 @@ def from_imod5_data(
         )
         simulation["ims"] = solution
 
-        # cleanup packages for validation
-        idomain = groundwaterFlowModel.domain
-        simulation.mask_all_models(idomain)
```

Comment:
there are a few examples on regridding. Even if  they don't need this masking, it may be helpful to modify them to do it anyway to make the reader aware  how to mask all the regridded packages

Reply:
The hondsrug example is the example where we regrid a simulation. In this example data is already made consistent, and regridding doesn't introduce any problems. I defer from adding unnecessary calls to the example, as it might lead users to blindly call computationally intensive functions. However, I want to create an example how to import a (problematic) iMOD5 model, where some cleanup needs to be done (for which cleanup utilities have to be made). We could introduce that here. I'll create a separate issue for that.

Reply:
See issue here https://github.com/Deltares/imod-python/issues/1164